src/proto/faabric.proto

```diff
@@ -130,40 +130,41 @@ message Message {
     int64 finishTimestamp = 19;
 
     bool isAsync = 20;
-    bool isPython = 21;
-    bool isStatusRequest = 22;
+    bool isStatusRequest = 21;
+    bool isThreaded = 22;
```

Comment:
I'm loathe to add another `isXYZ` flag, but I'll do another PR to move all the feature flags on a message into a `features` dictionary, i.e.

```
{
    "user": "foo",
    "function": "my_func",
    "features": {
          "threaded": true,
          "sgx": true,
          "exec_graph": true
    }
```